app/src/organisms/InterventionModal/StackerEmptyInterventionContent.tsx

```diff
@@ -0,0 +1,146 @@
+  ModuleLocation,
+} from '@opentrons/shared-data'
+import { Divider } from '/app/atoms/structure'
+import { css } from 'styled-components'
+import { InterventionCommandMessage } from './InterventionCommandMessage'
+
+const LABWARE_DESCRIPTION_STYLE = css`
+  flex-direction: ${DIRECTION_COLUMN};
+  grid-gap: ${SPACING.spacing8};
+  padding: ${SPACING.spacing16};
+  background-color: ${COLORS.grey20};
+  border-radius: ${BORDERS.borderRadius4};
+  @media ${RESPONSIVENESS.touchscreenMediaQuerySpecs} {
+    background-color: ${COLORS.grey35};
+    border-radius: ${BORDERS.borderRadius8};
+  }
+`
+
+const LABWARE_NAME_STYLE = css`
+  color: ${COLORS.grey60};
+  @media ${RESPONSIVENESS.touchscreenMediaQuerySpecs} {
+    ${TYPOGRAPHY.bodyTextBold}
+    color: ${COLORS.black90};
+  }
+`
+
+const DIVIDER_STYLE = css`
+  @media ${RESPONSIVENESS.touchscreenMediaQuerySpecs} {
+    display: ${DISPLAY_NONE};
+  }
+`
+
+const LABWARE_DIRECTION_STYLE = css`
+  align-items: ${ALIGN_CENTER};
+  grid-gap: ${SPACING.spacing4};
+  text-transform: ${TEXT_TRANSFORM_UPPERCASE};
+  @media ${RESPONSIVENESS.touchscreenMediaQuerySpecs} {
+    grid-gap: ${SPACING.spacing8};
+  }
+`
+
+const STACKER_IMAGE_STYLE = css`
+  flex-direction: ${DIRECTION_COLUMN};
+  grid-gap: ${SPACING.spacing8};
+  padding: ${SPACING.spacing16};
+  background-color: ${COLORS.grey35};
+  border-radius: ${BORDERS.lineBorder};
+`
+
+function stackerNameFormat(moduleLocation: ModuleLocation): string {
+  return 'Stacker ' + moduleLocation.slotName.charAt(0)
+}
+
+export interface StackerEmptyInterventionProps {
+  command: FlexStackerEmptyRunTimeCommand
+  analysis: CompletedProtocolAnalysis | null
+  run: RunData
+}
+
+export function StackerEmptyInterventionContent({
+  command,
+  analysis,
+  run,
+}: StackerEmptyInterventionProps): JSX.Element | null {
+  const { data: runCurrentState } = useRunCurrentState(run.id)
+  const flexStacker =
+    runCurrentState?.data.flexStackerStates[command.params.moduleId] ?? null
+
+  const analysisCommands = analysis?.commands ?? []
+  const labwareDefsByUri = getLoadedLabwareDefinitionsByUri(analysisCommands)
+
+  // Get the name of the labware to be removed from the stacker
+  let labwareName: string | null = null
+  if (flexStacker) {
+    const labwareDef = labwareDefsByUri?.[flexStacker.primaryLabwareURI] ?? null
+    labwareName = labwareDef?.metadata.displayName ?? null
+  }
+
+  const moduleLocation =
+    run?.modules.find(m => m.id === command.params.moduleId)?.location ?? null
+
+  if (
+    moduleLocation?.slotName == null ||
+    labwareName == null ||
+    flexStacker == null
+  )
+    return null
+  return (
+    <Flex
+      flexDirection={DIRECTION_COLUMN}
+      gridGap={SPACING.spacing12}
+      width="100%"
+    >
+      <Flex gridGap={SPACING.spacing32}>
+        <Flex
+          flexDirection={DIRECTION_COLUMN}
+          gridGap={SPACING.spacing12}
+          width="50%"
+        >
+          <Flex css={LABWARE_DESCRIPTION_STYLE}>
```

Comment:
https://www.figma.com/design/OIdG64Q5cgvEw82ish5Eee?node-id=12262-201923&main-component=1&state-group-id=10261-55562&fuid=979110576265559823
The designs show we should use `InterventionInfo` for this grey box instead of making a one-off. You can swap out this whole section with something like
```suggestion
            <InterventionInfo 
                layout="default" 
                type="location" 
                labwareName={labwareName}
                currentLocationProps={deckLabel={stackerNameFormat(moduleLocation)}
            />
```